net/ipv4/tcp_ipv4.c

```diff
@@ -1587,7 +1587,7 @@ struct sock *tcp_v4_syn_recv_sock(const struct sock *sk, struct sk_buff *skb,
 	 * so there is no appropriate security hook.
 	 */
 	if (tempesta_new_clntsk(newsk, skb)) {
-		tcp_v4_send_reset(newsk, skb);
+		ireq->aborted = true;
```

Comment:
Instead of introducing the new flag `aborted`, which is used in only one place, isn't it easier (and cleaner) to reuse `own_req` boolean return value?

Reply:
I think about it, but I don't agree with it, because if we reuse `own_req` we touch standard linux behavior. If own_req is false but sysct_abort_on_overflow is false we don't drop request sock.